docs/spec.md

```diff
@@ -2570,6 +2570,8 @@ produces a `result` tensor. Depending on the element type, does the following:
 // %result: [0.0, 1.71828187]
```

Comment:
update this example in spec.md to replace `f32` to `f64`. Also reflect the same same in the example presented in the description section of StableHLOOps.td file (As the spec.md and ODS should be in sync w.r.t to the example).